Examples/MagicWeather/MagicWeather/Constants.swift

```diff
@@ -16,13 +16,11 @@ struct Constants {
     /*
      The API key for your app from the RevenueCat dashboard: https://app.revenuecat.com
      */
-    #error("Modify this property to reflect your app's API key, then comment this line out.")
     static let apiKey = "api_key"
     
     /*
      The entitlement ID from the RevenueCat dashboard that is activated upon successful in-app purchase for the duration of the purchase.
      */
-    #error("Modify this property to reflect your app's entitlement identifier, then comment this line out.")
     static let entitlementID = "premium"
```

Comment:
why remove these two? the intent was that you'd always know you have to update them in order to use the sample app